skunkworks_crow/src/main/java/org/odk/share/views/ui/send/fragment/FilledFormsFragment.java

```diff
@@ -45,6 +49,7 @@
     public static final String INSTANCE_IDS = "instance_ids";
     private static final String INSTANCE_LIST_ACTIVITY_SORTING_ORDER = "instanceListActivitySortingOrder";
     private static final int INSTANCE_LOADER = 1;
+    private static final String[] TRANSFER_OPTIONS = {"via bluetooth", "via hotspot"};
```

Comment:
Lets create or resuse the string resources and `via` can be removed from the string.

Reply:
Fixed.